.coveragerc

```diff
@@ -739,6 +739,7 @@ omit =
     homeassistant/components/lutron/binary_sensor.py
     homeassistant/components/lutron/cover.py
     homeassistant/components/lutron/entity.py
+    homeassistant/components/lutron/event.py
```

Comment:
Tests would be nice, but that's for a separate PR.

Reply:
I've tried adding tests, but its a pita to mock the full library